src/modules/places/content/placeController.js

```diff
@@ -314,6 +314,26 @@ PlacesController.prototype.do_cmd_places_singleClickExpand = function() {
     }
 }
 
+PlacesController.prototype.is_cmd_places_singleClickOpen_enabled = function() {
```

Comment:
This should check that `ko.places` is ready/exists rather than just force it to be enabled. It's meant to check that it CAN run, not enable it.  Just a nit pick :).

Also note, you can do these steps using the new SDK: https://github.com/Komodo/KomodoEdit/blob/master/src/chrome/komodo/content/sdk/commands.js#L70

This is wicked @Defman21.  Learning the commands structure in Mozilla isn't fun.